meilisearch/src/option.rs

```diff
@@ -63,7 +63,7 @@ const DEFAULT_DUMP_DIR: &str = "dumps/";
 
 const MEILI_MAX_INDEXING_MEMORY: &str = "MEILI_MAX_INDEXING_MEMORY";
 const MEILI_MAX_INDEXING_THREADS: &str = "MEILI_MAX_INDEXING_THREADS";
-const DEFAULT_LOG_EVERY_N: usize = 100000;
+const DEFAULT_LOG_EVERY_N: usize = 100_000;
```

Comment:
I feel like we could remove this, as it doesn't appear to be used anywhere now. What do you think?

Reply:
It is used by milli in the indexer part to log for indexed documents.